test_3_add_up_numbers.py

```diff
@@ -2,8 +2,7 @@
 
 def add_up_numbers(arr):
   # Remove the `return None` code below and add your own code.
-  return None
-
+ return sum(int(x) for x in arr)
```

Comment:
Can you tell me more about the expression `int(x) for x in arr` and your thought process for including it?